bom/pom.xml

```diff
@@ -83,6 +83,11 @@
 				<artifactId>rdf4j-queryresultio-sparqljson</artifactId>
 				<version>${project.version}</version>
 			</dependency>
+			<dependency>
+				<groupId>org.eclipse.rdf4j</groupId>
+				<artifactId>rdf4j-queryresultio-sparqlstarjson</artifactId>
```

Comment:
Could we add the functionality to the existing sparql-json module, instead of introducing a new module? Same for the text one.

Reply:
Sure, what about turtlestar and trigstag then? I'd be happy to add those to turtle and trig as well instead of having them as separate modules.

Reply:
Good point. Yes, if you can. I think the functionality is closely enough related that it makes more sense to have this in the same module. Plus we're already a bit module-heavy in RDF4J, I'd rather avoid creating even more unless we have to.